package.json

```diff
@@ -36,21 +36,32 @@
     "url": "https://github.com/conventional-changelog/conventional-changelog.git"
   },
   "scripts": {
+    "clear": "pnpm -r --parallel --if-present clear && del ./coverage",
+    "build": "pnpm -r --parallel --if-present build",
     "lint": "eslint . --cache --cache-location ./node_modules/.cache/ --ignore-path .gitignore",
     "fix": "eslint . --fix --cache --cache-location ./node_modules/.cache/ --ignore-path .gitignore",
     "test:watch": "vitest watch",
     "test": "vitest run --coverage"
   },
   "version": "1.0.0",
   "devDependencies": {
+    "@trigen/eslint-config": "8.0.0-alpha.27",
+    "@types/node": "^20.5.9",
     "@vitest/coverage-v8": "^0.34.0",
     "better-than-before": "^1.0.0",
     "conventional-changelog-core": "workspace:^",
+    "del-cli": "^5.1.0",
     "eslint": "^8.0.0",
     "eslint-config-standard": "^17.0.0",
-    "forceable-tty": "^0.1.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^16.0.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^6.0.0",
+    "eslint-plugin-standard": "^4.1.0",
```

Comment:
q: why are you importing all those packages? 
esp, `eslint-plugin-node` as this package has been deprectated and has been replaced with `eslint-plugin-n`

Reply:
sorry, invalid rebase

Reply:
```
 WARN  Issues with peer dependencies found
.
└─┬ eslint-config-standard 17.1.0
  ├── ✕ missing peer eslint-plugin-import@^2.25.2
  ├── ✕ missing peer eslint-plugin-n@"^15.0.0 || ^16.0.0 "
  └── ✕ missing peer eslint-plugin-promise@^6.0.0
  ```